src/asm/lexer.c

```diff
@@ -731,7 +731,9 @@ static uint32_t yylex_NORMAL(void)
 			return T_STRING;
 		} else if (*pLexBuffer == '{') {
 			pLexBuffer++;
-			yylex_ReadBracketedSymbol(yylval.tzString, 0);
+			size_t len = yylex_ReadBracketedSymbol(yylval.tzString,
+							       0);
+			yylval.tzString[len] = 0;
```

Comment:
Is there any check to make sure this sting doesn't overflow the array ``tzString``?

Reply:
Yes. I explained why it won't overflow in one of the comments above. https://github.com/rednex/rgbds/pull/365#issuecomment-506108527

Reply:
Oh, sorry, you're right.